crates/core/src/db/relational_db.rs

```diff
@@ -1376,6 +1376,23 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_row_count() -> ResultTest<()> {
```

Comment:
@Shubham8287 I think @gefjon is suggesting to add a duplicate insert and a delete in either this test or a separate one.

Reply:
I added them here already with other similar tests - https://github.com/clockworklabs/SpacetimeDB/pull/1242/files#diff-2e3e28b72d24c870151676e95eadbbcf99aa90580a9023cf85d3b6244be0ed23R1160